mandatory/3-playing-computer.js

```diff
@@ -14,29 +14,33 @@
   6. What value does the "a" parameter take in the first "f1" call?
   7. What is the value of the "a" outer variable when "f1" is called for the first time?
 */
-
 let x = 2;
 let a = 6;
 
 const f1 = function (a, b) {
+  //b is undefined 
   return a + b;
-};
+}
 
 const f2 = function (a, b) {
+  //b is undefined and adds x to the sum of a and b
   return a + b + x;
-};
+}
 
 console.log(x);
 console.log(a);
 console.log(b);
 
 for (let i = 0; i < 5; ++i) {
   a = a + 1;
+  //a becomes 7
   if (i % 2 === 0) {
+    //this is going to call the function f2, two times, when i === 2 && i ===4
     const d = f2(i, x);
     console.log(d);
   } else {
+    // this is going to call the function f1, 3 times, when i === 0 && i === 1 && i ===3
```

Comment:
this is wrong because  0 divided by any number is 0, so when i === 0 then f2 will execute not f1. (0 divided by 2 = 0)

Reply:
Nice catch, I understand it now... 0 divided by any number is 0